client/src/components/UI/card/JobCard.tsx

```diff
@@ -0,0 +1,68 @@
+import { Card, Avatar, Button } from "antd";
+import { IconExternalLink } from "@tabler/icons-react";
+import styling from "./JobCard.module.css";
+
+interface JobCardProps {
+  name?: string;
+  company_name?: string;
+  company_location?: string;
+  description?: string;
+  avatarUrl?: string;
+  employees?: string;
+  level?: string;
+  match?: string;
+  location?: string;
+  department?: string;
+}
+
+const JobCard: React.FC<JobCardProps> = ({
+  name,
+  company_name,
+  company_location,
+  description,
+  avatarUrl,
+  employees,
+  level,
+  match,
+  location,
+  department,
+}) => {
+  const truncatedDescription: string | undefined = description
+    ? typeof description === "string" && description.length > 370
+      ? `${description.slice(0, 370)}...`
+      : description
+    : undefined;
+
+  return (
+    <Card className={styling.card}>
+      <div className={styling["card-content"]}>
+        <div className={styling["job-header"]}>
+          <Avatar className={styling.avatar} src={avatarUrl} />
+          <div className={styling["company-info"]}>
+            <h4 className={styling["company-name"]}>{company_name}</h4>
+            <h5 className={styling["company-location"]}>{company_location}</h5>
+            <h6 className={styling.employee}>{employees} employees</h6>
```

Comment:
Better to use `p` instead of all these `h4`, `h5`, and `h6`. As a not written rule, we only use `h1`, `h2`, and `h3`.

Reply:
ok.why should i use p? what for are h4,h5,h6 tags?Could you explain a little bit i found these approach interesting.thank you for the tips

Reply:
It is not wrong to use  `h4`, `h5`, and `h6` at all, and they might be used for a specific design element, however in web development it is a rare case when we would need to use more than 4 font size styles, is for that we normally use `h1` to `h3` and `p`. Also, it makes it easier to read when you don't have so many different elements.